gnat2goto/driver/arrays.adb

```diff
@@ -228,24 +227,28 @@ package body Arrays is
          while Present (Pos_Iter) loop
             declare
                Expr : constant Irep := Do_Expression (Pos_Iter);
+               Pos_Constant : constant Irep :=
```

Comment:
This seems to be duplicating some logic from the initialisation code, is there some opportunity to generalise?